### DISCUSSION TITLE
How to change the gray background color when button is disabled

### DISCUSSION THREAD

**Discussion by shamxeed** [callstack/react-native-paper#2624] [Q&A] [answered]
I want to change the default gray background when the button is disabled. I thought that could be archive using theme prop but to no avail.

What I tried:
```jsx
<Button
   theme={{colors: {disabled: 'red'}}
>
    Press Me
<Button>
```

Any idea on how to archive that?

**Comment by raajnadar** [comment, marked as answer]
https://github.com/callstack/react-native-paper/issues/2625
